snuba/subscriptions/data.py

```diff
@@ -72,6 +72,19 @@ def from_string(cls, value: str) -> SubscriptionIdentifier:
 class SubscriptionData:
     """
     Represents the state of a subscription.
+    project_id, resolution, time_window are all specified as separate fields
+    in the subscription payload by the user, alongside the query. This allows
+    us to pre-build one subscription query and vary these as separate parameters.
+
+    time_window_sec becomes part of the query condition (i.e the WHERE), and the
+    subscription query will look at the past 'x' seconds (as specified by the
+    window) of events.
+
+    project_id becomes part of the query condition, and the query will filter
+    records by matching on the specified id.
+
+    resolution_sec is used to determine when the scheduler creates tasks
+    for the executor to run.
```

Comment:
Would this docstring be better suited on the create subscription endpoint or in usage documentation instead of on this class? If it aims to help users to create and use subscriptions more easily I think this is buried deep inside the subscriptions implementation.

Reply:
This is a good point. I'll move it